README.md

```diff
@@ -5,78 +5,13 @@
 A set of services that handle reading and writing data from Postgres about
 compute resources.
 
-## Polylith
-This project follows the [Polylith Architecture]() using the [Polylith Poetry
-Plugin](https://davidvujic.github.io/python-polylith-docs/installation/).
-
-[Installation
-instructions](https://davidvujic.github.io/python-polylith-docs/installation/)
-for the plugin.
-
-Use `poetry poly info` to get an overview of the contained
-components/bases/projects. Refer to the documentation of the plugin for further
-details.
 
 ## Initial Setup
 
 1. `poetry install`
 2. `pre-commit install` to install pre commit hooks
-3. `make run` to run the server
+3. `DUMMY_STORES=true poetry run python bases/renku_data_services/data_api/main.py --debug --dev --fast`
```

Comment:
```suggestion
3. `DUMMY_STORES=true poetry run python bases/renku_data_services/data_api/main.py --debug --dev`
```

We dont really need `--fast` when developing. And this command is only for developing since we have `DUMMY_STORES=true`

Reply:
this was just a borked merge conflict resolution, I've changed it back to `make run`